openmc/data/multipole.py

```diff
@@ -112,7 +112,8 @@ def _broaden_wmp_polynomials(E, dopp, n):
         erf_beta = 1.0
         exp_m_beta2 = 0.0
     else:
-        erf_beta = np.erf(beta)
+        from scipy.special import erf
```

Comment:
I found this `erf()` in the `else` branch is not always called so made the `import` inside. If making it outside is better, we have another choice: [math package also provides this error function ](https://docs.python.org/3/library/math.html#math.erf).